apps/lfx-one/src/app/shared/components/table/table.component.scss

```
@@ -100,11 +100,19 @@
         margin-left: auto;
         font-size: 0.875rem !important;
         color: #6b7280;
+        // Keep the "Showing X to Y of Z" report on a single line so the rpp
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)